### PR TITLE
Ds ajax photogram

### PR DESCRIPTION
**Pull request by dylansarikas**
adds the different parts for ajax photogram

### CODE REVIEW

File: app/controllers/comments_controller.rb
Comment (APatel-AI) [inline app/controllers/comments_controller.rb:1]: Looks good.

File: app/views/comments/_comment.html.erb
Comment (APatel-AI) [inline app/views/comments/_comment.html.erb:1]: Great Job

File: app/views/comments/create.js.erb
Comment (APatel-AI) [inline app/views/comments/create.js.erb:1]: Great Job in using a variable. The code is well structured.

**Review by APatel-AI — APPROVED** [review]
Great Job Dylan!!. The code is well structured and concise.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -59,6 +64,10 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb]: nitpik: please be mindful of unnecessary white spaces

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var added_comment = $("<%= j(render @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/create.js.erb:1]: I love that you used variables to make your code readable

File: app/views/follow_requests/_follow_unfollow.html.erb [hunk 3]
```diff
@@ -1,17 +1,19 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+<div class="<%=dom_id(recipient)%>_buns">
```
Comment (ahuynh3a) [inline app/views/follow_requests/_follow_unfollow.html.erb]: I noticed that you did not include the sender id in your class, but you passed in a sender id in your create.js.erb and destroy.js.erb. Without the the dom_id for sender, your code will only work for one person and nobody else. Could you add the dom_id for the sender here and update your create.js.erb as well as your destroy.js.erb?

**Review by ahuynh3a — CHANGES_REQUESTED** [review]
Hi Dylan! You are on the right track all of your code works except for follow/unfollow. Could you go in and make the changes I stated, I can approve after

File: app/views/follow_requests/_follow_unfollow.html.erb (hunk 3 above)
Comment (dylansarikas) [reply app/views/follow_requests/_follow_unfollow.html.erb]: changed!

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (dylansarikas) [reply app/controllers/comments_controller.rb]: changed!

**Review by ahuynh3a — APPROVED** [review]
Thank you for making the changes! Great job!

**Pull request merged**